src/core/eme/session.ts

```diff
@@ -268,38 +279,17 @@ function createSession(
       // If that was the case though, we should subscribe here.
       $loadedSessions.deleteAndClose(session);
       $storedSessions.delete(initData);
```

Comment:
This is not super evident to perform cache cleaning here, as it's nowhere near the cache-creating logic. We should find a better solution (this is just a thought I have now)

Reply:
Also I'm not sure the comment is still valid here (I'm not even sure of what it was for)